data/lua/.library/infoboxes.lua

```diff
@@ -0,0 +1,83 @@
+InfoBoxStart = {} --Game.Client.LocalTime
+InfoBoxDuration = 3 -- x in seconds
+InfoBoxHeader = {} -- displayed header
+InfoBoxText = {} -- displayed Text in the box
+InfoBoxMode = {} -- 1 = creating, 2 = drawing_normal, 3 = deleting
+InfoBoxRectHeader = {}
+InfoBoxRectText = {}
+InfoBoxMaxWidth = 200
+InfoBoxMaxHeight = 100
+InfoBoxHeightOffset = 10
+InfoBoxFadeunits = 1 -- +- 1 Pixels per draw!
+InfoBoxMaxFPS = Game.Client.FPS
```

Comment:
the fps can change at any time, so rather take the current fps into calculation instead of the maximum